.github/workflows/unit-tests.yml

```diff
@@ -14,6 +14,8 @@ on:
 jobs:
   # This workflow contains a single job called "build"
   build:
+    env:
+      ACTIONS_ALLOW_UNSECURE_COMMANDS: true
```

Comment:
Why this change?

Reply:
You need to use this to set env variables (https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#environment-files). It's a recent change to github actions which broke the CI flow.